### ISSUE TITLE
Searching with spaces results in error

### ISSUE THREAD

**Issue by svenvanderburg** [fair-workflows/nanopub#112]
This returns 2 results:
```python
>>> results = client.find_nanopubs_with_text('Picoides')
>>> for result in results:
>>>    print(result)
{'np': 'http://purl.org/np/RAMzGwLotMRSQTHRCS15B6hvuYSyEupGnxZaFU3EmcItA', 'description': 'Inter-species interaction of Picoides villosus with Ips', 'date': '2020-12-24T10:51:43.931Z'}
{'np': 'http://purl.org/np/RAMzGwLotMRSQTHRCS15B6hvuYSyEupGnxZaFU3EmcItA', 'description': 'Picoides villosus', 'date': '2020-12-24T10:51:43.931Z'}
```

But this gives an error:
```python
>>> results = client.find_nanopubs_with_text('Picoides villosus')
```

* [ ] Investigate whether this is on our side or nanopub server side
* [ ] Raise issue server side or fix client side

**Comment by svenvanderburg** [comment]
It seems like it is client side, see results for: http://grlc.nanopubs.lod.labs.vu.nl/api/local/local/find_nanopubs_with_text?text=Picoides%20vilosus

**Comment by svenvanderburg** [comment]
There was already an issue for this: https://github.com/peta-pico/nanopub-api/issues/2. 

We should be able to solve this on our side. I see 2 solutions:
1. by always using single quotation marks we only support a simple text search (which is the behavior I expected)
2. add detailed information on how this works to the documentation

I would vote for 1, because I don't think there is any documentation for 2. Or is there @tkuhn ?

**Comment by tkuhn** [comment]
Yes, you need to add an "AND" like "Picoides villosus" or quotes around it. This is directly passed to the Virtuoso full-text search, so it's probably documented there.

**Comment by raar1** [comment]
I also vote for 1, because it's the behaviour I would have expected. We can provide a different method for composing the Virtuoso style search string, although I would rather avoid something that is too dependent on the choice of triple store on the backend (will it always be virtuoso for every server?)
